tutorials/beginner/logs.md

```diff
@@ -6,11 +6,22 @@ layout: tutorial-beginner
 
 # Looking to the logs
 
-While using openHAB, we will need to look the the logs generated by the software.
-
-Two log files are going to interest us.
+While using openHAB, you may need to look at the logs generated by the software to see if there are any issues that need your attention.
+
+Two log files are going to interest us which are openhab.log and events.log 
+The events.log file contains what is happening on the Openhab event bus like sensors changing states, very handy for writting rules as you can see things changing in realtime. 
+The other file openhab.log shows the output of all bindings and addons and allows you to fault find issues with a binding. 
+Before asking for help on the forum, it is worth checking openhab.log for clues as to what is causing any issues as this will result in getting help much faster.
+
+To watch the logs in realtime with a openhabian based setup use the below linux command which can be done via SSH with a program called putty from a windows or mac machine. 
+CTRL+C will close the stream. 
+You can also use SAMBA/network shares to open or copy the file directly.
+```text
+tail -f /var/log/openhab2/openhab.log -f /var/log/openhab2/events.log
+```
```

Comment:
Please add a newline between the the code fences and the text.
(Needed for generating valid html files out of this markdown file.)